bc-rest/src/main/java/org/jboss/da/bc/model/BuildConfiguration.java

```diff
@@ -0,0 +1,79 @@
+package org.jboss.da.bc.model;
```

Comment:
These entities are different. BC in communication is PNC BC, but this BC is our model for BC import process in PNC. The entities are slightly different